server/router.js

```diff
@@ -0,0 +1,25 @@
+// require express, express.router
+const express = require('express');
+const router = express.Router();
+
+// make timeLog function that sends back time
+var timeLog = (req, res) => {
+  var time = `${Date.now()}`;
+  res.send(time);
+}
+
+// make test route for GET req
+router.get('/', (req, res) => {
+  res.status(200).send('Proxy api is working!');
+});
+
+// make test route to time
+router.get('/time', timeLog);
```

Comment:
Btw, you don't need to have routes to handle these endpoints - since that's handled within your module